assets/pages/paper-subscription-landing/paperSubscriptionLandingPageReducer.js

```diff
@@ -2,22 +2,61 @@
 
 // ----- Imports ----- //
 
+import { combineReducers } from 'redux';
 import type { CommonState } from 'helpers/page/commonReducer';
-import { type PaperBillingPlan } from 'helpers/subscriptions';
+import { type Option } from 'helpers/types/option';
 import { getQueryParameter } from 'helpers/url';
+import { type PaperBillingPlan } from 'helpers/subscriptions';
 import { ProductPagePlanFormReducerFor, type State as FormState } from 'components/productPage/productPagePlanForm/productPagePlanFormReducer';
 
+export type PaperPrices = {
```

Comment:
I think I would probably model this as:
```
PaperPrices = {
    collection {
        everyday: Option<number>,
        sixday: Option<number>,
        weekend: Option<number>,
        sunday: Option<number>,
    }
    ...
```

Reply:
that is cleaner but both the dataset fetching and setting a default ticked option from the url rely on the price list being flat, it's a pretty involved change for this PR :(